browser/app/js/buckets/Bucket.js

```diff
@@ -94,4 +100,16 @@ export class Bucket extends React.Component {
   }
 }
 
-export default Bucket
+const mapStateToProps = state => {
```

Comment:
I don't think we are using `sidebarOpen` anywhere in this file so we can remove the `mapStateToProps` here and change line 115 to `export default connect(undefined, mapDispatchToProps)(Bucket)`